src/tests/containerizer/cgroups2_tests.cpp

```diff
@@ -115,6 +119,48 @@ TEST_F(Cgroups2Test, ROOT_CGROUPS2_AssignProcessToCgroup)
   AWAIT_EXPECT_WTERMSIG_EQ(SIGKILL, process::reap(pid));
 }
 
+
+TEST_F(Cgroups2Test, ROOT_CGROUPS2_DeviceController)
+{
+  namespace devices = cgroups2::devices;
+
+  ASSERT_SOME(cgroups2::create(TEST_CGROUP));
+  string absolutePath = cgroups2::path(TEST_CGROUP);
+
+  // Deny all access.
+  Try<devices::Entry> entry = devices::Entry::parse("c *:* rwm");
+  ASSERT_SOME(entry);
```

Comment:
would be good to get more coverage of the logic here, rather than only testing a single deny all character devices?